_includes/specials/taproot/en/10-ln-with-taproot.md

```diff
@@ -0,0 +1,198 @@
+*By [ZmnSCPxj][]*
+
+In this post, we'll look at two privacy features that [taproot][topic taproot] enables
+for LN:
+
+* [PTLCs][topic ptlc] over LN.
+* P2TR Channels.
+
+### PTLCs Over LN
+
+PTLCs enable [many features][suredbits payment points], with a major
+feature for LN being [payment decorrelation][p4tr ptlcs] without
+any need to randomize routes.[^route-randomization] Every node along a
+single-path or [multipath][topic multipath payments] route can be given a scalar that is used to
+tweak each forwarded PTLC, enabling *payment decorrelation* where
+individual forwards no longer leak the unique identifier for each
+LN payment.
+
+PTLCs are ***not a privacy panacea***.  If a surveilant node sees a
+forward with a particular timelock and value, and a second surveilant
+node shortly after sees a forward with a *lower* timelock and *slightly
+lower* value, then *very likely* those forwards belong to the same
+payment path, even if the surveilant nodes can no longer correlate them
+via a unique identifying hash.  However, we *do* get:
+
+* Increased uncertainty in the analysis.  The probabilities surveilant
+  can work with are now lower and thus their information is that much
+  less valuable.
+* A *lot* more decorrelation in multipath payments.  Separate paths will
+  not have strong timelock and value correlation with each other, and if
+  LN succeeds, there should be enough payments that timing
+  correlation is not reliable either.
+* No increase in cost compared to an [HTLC][topic htlc] (and possibly even a slight
+  cost reduction due to [multisignature efficiency][p4tr
+  multisignatures]).
+
+In principle, a pre-taproot channel can be upgraded to support PTLCs without
+closing and reopening the channel.  Existing channels can host PTLCs by
+creating an offchain transaction that spends the existing non-taproot
+funding output to a taproot output containing a PTLC.  Thus getting
+support for PTLCs over LN does not require any cost to users
+beyond each node and its channel peers upgrading their software.
+
+However, to actually use PTLCs, *every* forwarding node from the spender
+to the receiver must support PTLCs.  This means PTLC support may remain
+largely unused until a sufficient number of nodes have upgraded.  They
+don't all necessary need to use the same protocol (there could be
+multiple PTLC protocols), but they all must support some PTLC protocol.
+Having to support multiple PTLC protocols would be an added maintenance
+burden and I *hope* we do not have too many such protocols (ideally just
+one).
+
+### P2TR Channels
+
+One solution for improving the decorrelation between the base layer and
+the LN layer has been [unpublished channels][topic unannounced channels]---channels whose
+existence isn't gossiped on LN.
+
+Unfortunately, every LN channel is a 2-of-2, and in the current
+pre-taproot Bitcoin, every 2-of-2 is *openly* coded.  LN is the
+most popular user of 2-of-2 multisignature, so any blockchain explorer
```

Comment:
Optech style guide is "block chain"

```suggestion
most popular user of 2-of-2 multisignature, so any block chain explorer
```

(there are 3 other "blockchain" instances in the document)